Libraries/src/Amazon.Lambda.Logging.AspNetCore/LambdaILogger.cs

```diff
@@ -45,30 +43,33 @@ public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Except
             }
 
             // Format of the logged text, optional components are in {}
-            //  {[LogLevel] }{Category: }{EventId: }MessageText {Exception}{\n}
+            //  {[LogLevel] }{ => Scopes : }{Category: }{EventId: }MessageText {Exception}{\n}
 
             var components = new List<string>(4);
             if (_options.IncludeLogLevel)
             {
                 components.Add($"[{logLevel}]");
             }
+
+            GetScopeInformation(components, multiLine: _options.IncludeNewline);

```

Comment:
I'm seeing scopes always being included regardless of what `IncludeScopes` is set to. Seems like we need an if check before calling GetScopeInformation.

Reply:
sorry, I've missed it.
I based my implementation on https://github.com/aspnet/Extensions/blob/642b9263abe16ba1cd038f27c8cb4661763ff5bb/src/Logging/Logging.Console/src/ConsoleLogger.cs#L293 but missed that condition

Reply:
it would not log scopes even without the check, since Scopes would be set to `NullExternalScopeProvider`
I think it's more readable with the check like you suggested